latexindent.pl

```diff
@@ -269,9 +269,32 @@
 
 # get information about user settings- first check if indentconfig.yaml exists
 my $indentconfig = File::HomeDir->my_home . "/indentconfig.yaml";
+
+if ( -e $indentconfig)
+{
+      print $logfile "Using settings file $indentconfig\n";
+      if ( -e File::HomeDir->my_home . "/.indentconfig.yaml"){
+        print $logfile File::HomeDir->my_home,"/.indentconfig.yaml has been found, but $indentconfig takes priority\n";
+      }
+      else
+      {
+        print $logfile "Alternatively, ",File::HomeDir->my_home,"/.indentconfig.yaml can be used\n";
+      }
+}
+else
+{
+      $indentconfig = File::HomeDir->my_home . "/.indentconfig.yaml";
+      if ( -e $indentconfig)
+      {
+        print $logfile "Using settings file $indentconfig\n";
+        print $logfile "Alternatively, ",File::HomeDir->my_home,"/indentconfig.yaml can be used\n";
+      }
+}
+
 if ( -e $indentconfig and !$onlyDefault )
 {
-      print $logfile "Reading path information from ",File::HomeDir->my_home,"/indentconfig.yaml\n";
+      print $logfile "Reading path information from $indentconfig\n";
+      print $logfile "Only default settings requested, not reading USER settings from $indentconfig\n";
```

Comment:
Thanks! I'll do some more testing this week; will touch base with you later
in week.

On Monday, April 13, 2015, Jacobo Díaz <notifications@github.com> wrote:

> In latexindent.pl
> <https://github.com/cmhughes/latexindent.pl/pull/23#discussion_r28273946>:
>
> >  if ( -e $indentconfig and !$onlyDefault )
> >  {
> > -      print $logfile "Reading path information from ",File::HomeDir->my_home,"/indentconfig.yaml\n";
> > +      print $logfile "Reading path information from $indentconfig\n";
> > +      print $logfile "Only default settings requested, not reading USER settings from $indentconfig\n";
>
> Done. Thanks!
> … <#14cb4665d0897ac1_>
> On Mon, Apr 13, 2015 at 10:04 PM, cmhughes <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote: In
> latexindent.pl <
> https://github.com/cmhughes/latexindent.pl/pull/23#discussion_r28273485>:
> > if ( -e $indentconfig and !$onlyDefault ) > { > - print $logfile "Reading
> path information from ",File::HomeDir->my_home,"/indentconfig.yaml\n"; > +
> print $logfile "Reading path information from $indentconfig\n"; > + print
> $logfile "Only default settings requested, not reading USER settings from
> $indentconfig\n"; I don't think you wanted this line here :) This should
> only be used if $onlyDefault is used; can you delete it? — Reply to this
> email directly or view it on GitHub <
> https://github.com/cmhughes/latexindent.pl/pull/23/files#r28273485>.
>  -- *Jacobo Díaz García* Profesor Titular de Universidad ETS de
> Ingenieros de Caminos, Canales y Puertos Universidad de La Coruña Campus de
> Elviña - 15071 - La Coruña 981 167 000 - ext 5478
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cmhughes/latexindent.pl/pull/23/files#r28273946>.
>